src/code/z_player_lib.c

```diff
@@ -1076,132 +1148,93 @@ void Player_DrawGetItem(GlobalContext* globalCtx, Player* this) {
     }
 }
 
-void func_80090A28(Player* this, Vec3f* vecs) {
-    D_8012608C.x = D_80126080.x;
+void Player_GetSwordTip(Player* this, Vec3f* swordTip) {
+    sSwordTipPos2.x = sSwordTipPos1.x;
 
     if (this->unk_845 >= 3) {
         this->unk_845 += 1;
-        D_8012608C.x *= 1.0f + ((9 - this->unk_845) * 0.1f);
+        sSwordTipPos2.x *= 1.0f + ((9 - this->unk_845) * 0.1f);
     }
 
-    D_8012608C.x += 1200.0f;
-    D_80126098.x = D_8012608C.x;
+    sSwordTipPos2.x += 1200.0f;
+    sSwordTipPos3.x = sSwordTipPos2.x;
 
-    Matrix_MultVec3f(&D_80126080, &vecs[0]);
-    Matrix_MultVec3f(&D_8012608C, &vecs[1]);
-    Matrix_MultVec3f(&D_80126098, &vecs[2]);
+    Matrix_MultVec3f(&sSwordTipPos1, &swordTip[0]);
+    Matrix_MultVec3f(&sSwordTipPos2, &swordTip[1]);
+    Matrix_MultVec3f(&sSwordTipPos3, &swordTip[2]);
 }
 
-void func_80090AFC(GlobalContext* globalCtx, Player* this, f32 arg2) {
-    static Vec3f D_801260C8 = { -500.0f, -100.0f, 0.0f };
-    CollisionPoly* sp9C;
+void Player_DrawHookshotReticle(GlobalContext* globalCtx, Player* this, f32 arg2) {
+    static Vec3f posOffset = { -500.0f, -100.0f, 0.0f };
+    CollisionPoly* polygon;
     s32 bgId;
-    Vec3f sp8C;
-    Vec3f sp80;
-    Vec3f sp74;
-    Vec3f sp68;
-    f32 sp64;
-    f32 sp60;
+    Vec3f posA;
+    Vec3f posB;
+    Vec3f aimProjectedPos; // position that the hookshot is aimed towards
```

Comment:
I think "projected" gives the wrong idea (at first I thought it had to do with the projection matrix)

I think just `aimPos` is fine